apps/experiments/models.py

```diff
@@ -747,6 +747,13 @@ def compare_with_latest(self):
             version.compare(prev_version.version)
         return version.fields_changed
 
+    def archive_working_experiment(self):
+        if self.is_working_version:
+            if self.has_versions:
+                for version in self.versions.all():
```

Comment:
This can be done with a single query. Something like

```python
from field_audit.models import AuditAction

Experiment.objects.filter(working_version=self).update(is_archived=True, audit_action=AuditAction.AUDIT)
```